devel/openssl/Portfile

```diff
@@ -6,7 +6,7 @@ PortGroup           openssl 1.0
 name                openssl
 epoch               2
 version             [openssl::default_branch]
-revision            18
+revision            19
 
 categories          devel security
 platforms           darwin
```

Comment:
Maybe drop this line per the lint warning.

Reply:
Update runtime image of aspnet dotnet to 8.0.8-jammy worked for myself